website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -9,6 +9,10 @@ description: >-
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+## 2023-08-08
+* Update [Create a Policy Set](/terraform/cloud-docs/api-docs/policy-sets#create-a-policy-set) and [Update a Policy Set](/terraform/cloud-docs/api-docs/policy-sets#update-a-policy-set) API endpoints to include a `projects` relationship
```

Comment:
nitpick - add . to be consistent

Reply:
These 2 endpoints already exist so we're updating the existing ones, not adding new ones. The attach and detach endpoints are brand new so we are adding them, not updating them.